### DISCUSSION TITLE
3.6.2版想做测试,但是 @tarojs/components/dist-h5/react 找不到了

### DISCUSSION THREAD

**Discussion by adamgogogo** [NervJS/taro#13631] [Q & A]
这个目录迁移到哪里呢？ @tarojs/components-react ？

**Comment by TheKonka** [comment]
@tarojs/components/dist 就是吧？

**Reply by adamgogogo** [reply]
> @tarojs/components/dist

默认的 @tarojs/components 会报错，`Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined.`

还是要设置moduleNameMapper才行

**Comment by adamgogogo** [comment]
搞定，换成react就可以了...
